.github/workflows/land-blocking.yml

```diff
@@ -37,28 +37,32 @@ jobs:
         if: steps.check_ks.outputs.should_run == 'true'
         # NOTE Remember to update PR comment payload if cti cmd is updated.
         run: |
+          CTI_OUTPUT_LOG=$(mktemp) ./scripts/cti \
+            --k8s \
+            --tag ${TEST_TAG} \
+            --report report-k8s.json \
+            --run bench \
+            --k8s-fullnodes-per-validator=0 \
+            --k8s-num-validators=100 &> k8s.log &
           export CTI_OUTPUT_LOG=$(mktemp)
           echo "::set-env name=CTI_OUTPUT_LOG::$CTI_OUTPUT_LOG"
-          if [[ ${USE_K8S} == true ]]; then
+          JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
             ./scripts/cti \
-              --k8s \
-              --marker land \
-              --tag ${TEST_TAG} \
-              --report report.json \
-              --run bench \
-              --k8s-fullnodes-per-validator=0 \
-              --k8s-num-validators=100
-          else
-            JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
-              ./scripts/cti \
-              --marker land \
-              --tag ${TEST_TAG} \
-              --report report.json \
-              --run bench
-          fi
+            --marker land \
+            --tag ${TEST_TAG} \
+            --report report.json \
+            --run bench
+          return_code=$?
           echo "report.json start"
           cat report.json
           echo "report.json end"
+          echo "report-k8s.json start"
+          cat report-k8s.json
+          echo "report-k8s.json end"
+          echo "k8s.log start"
+          cat k8s.log
+          echo "k8s.log end"
+          exit $return_code
```

Comment:
Frankly, this feels like a clutter ATM.  IIUC, the goal is to dry-run the k8s option for some period in the background in order to catch edge cases in infra.  With that, we need to be able to detect and look into failures in k8s result.  I'd imagine it'd be hard to scroll through all the output logs.  Any thoughts?

Reply:
My plan is to monitor the success/failure of the k8s jobs launched by gh-actions by looking at the pod statuses on all the cluster-test k8s clusters.

So basically running `kubectl get pods -l app=cluster-test` on all the clusters and making sure they all pass.